tests/test.sh

```diff
@@ -13,6 +13,8 @@ pycodestyle ../setup.py --max-line-length=100
 [[ $? -ne 0 ]] && exit 1
 pycodestyle scripts/run_tests.py --max-line-length=100 --ignore=W503,E501,E402
 [[ $? -ne 0 ]] && exit 1
+pycodestyle scripts/release_test.py --max-line-length=100
+[[ $? -ne 0 ]] && exit 1
```

Comment:
nit: I think this can be one-lined by `...-length=100 || exit 1`.

Reply:
Thanks. I might change this at a later point.